Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "Apache-2.0"
 name = "mongodb"
 readme = "README.md"
 repository = "https://github.com/mongodb-labs/mongo-rust-driver-prototype"
-version = "0.3.3"
+version = "0.3.4"
```

Comment:
Accidentally didn't push this to master before opening the PR, but it's pushed now